logfire/_internal/async_.py

```
@@ -120,3 +123,119 @@ def _callback_attributes(callback: Any) -> _CallbackAttributes:
+    def _log_pause(self, duration: float, stack: list[StackInfo]) -> None:
+        try:
+            attributes: _CallbackAttributes = {'stack': stack}
+            if stack:  # pragma: no branch
+                # The innermost frame is where the loop thread was blocked when sampled.
+                attributes = {**stack[-1], **attributes}
+            self.logfire.warn(
+                'Event loop blocked for {duration:.3f} seconds',
+                duration=duration,
+                **attributes,
+            )
+        except Exception:  # pragma: no cover
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)